package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "scripts": {
+    "pretest": "forever stopall -s && forever start -s ./node_modules/.bin/http-server",
+    "test": "grunt test && pa11y localhost:8080/demo -s WCAG2AA && npm run coverage",
+    "posttest": "forever stopall -s",
+    "coverage": "./coverage.sh"
+  },
+  "devDependencies": {
+    "cf-component-demo": "^1.0.1",
+    "cf-grunt-config": "^1.0.1",
+    "coveralls": "^2.11.3",
+    "forever": "^0.15.1",
+    "glob": "~5.0.14",
+    "grunt": "~0.4.5",
+    "http-server": "^0.8.0",
+    "istanbul": "git://github.com/gotwarlost/istanbul#harmony",
+    "jasmine-node": "~1.14.5",
+    "jit-grunt": "~0.9.1",
+    "jsx-test": "0.7.5",
+    "load-grunt-tasks": "~3.2.0",
+    "node.extend": "~1.1.5",
+    "pa11y": "^2.4.2",
+    "phantomjs": "^1.9.18",
+    "react": "0.13.3",
```

Comment:
It's there because I uh... never removed it from the package.json?

Honestly, I don't know how many of those are actually needed because I used a dummy CF repo that @ascott1 made. Several are there for testing, but there might actually be more that can be removed. I'm not familiar enough with every aspect of CF to know if I can remove others safely...